sources/platform/actors/publishing/badge.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Actor Status Badge
+description: Apify has a QA system that regularly runs automated tests to ensure that all Actors in the store are functional.
+slug: /actors/publishing/status-badge
+sidebar_position: 4
+---
+
+Apify Actor Status Badge can be embedded in 3rd party documentation to show current health status and engagement of the Actor in the Apify Platform.
+
+---
+
+### Actor Status Badge Example
+
+This is the badge generated for the [Apify's Website Content Crawler](https://apify.com/apify/website-content-crawler Actor).
+
+[![Website Content Crawler Actor](https://apify.com/actor-badge?actor=apify/website-content-crawler)](https://apify.com/apify/website-content-crawler)
+
+
+### How to embed the Actor Status Badge
+
+The Badge is a dynamic SVG image loaded from the Apify platform. The Badge is served from the URL Template:
+
+```text
+https://apify.com/actor-badge?actor=:username/:actor-name
+```
+
+In order to embed the badge in the HTML documentation, just use it as an image wrapped in a link as shown in the example below. Don't froget to use the `username` and `actor-name` of your Actor.
+
+#### HTML Example
+
+```html
+<a href="https://apify.com/apify/website-content-crawler">
+  <img src="https://apify.com/actor-badge?actor=apify/website-content-crawler">
+</a>
+```
+
+#### Markdown Example
+
+```markdown
+[![Website Content Crawler Actor](https://apify.com/actor-badge?actor=apify/website-content-crawler\)](https://apify.com/apify/website-content-crawler)
+```
```

Comment:
Instead of making these separate headings we can utilize [Docusaurus tabs](https://github.com/apify/apify-docs?tab=readme-ov-file#code-tabs)  and have this be one heading "Examples"

Reply:
Resolved in e5ab63382fc27e929ddf261ec1c96812181aa6a5